tensorflow/python/estimator/canned/dnn.py

```diff
@@ -40,6 +41,27 @@
 _LEARNING_RATE = 0.05
 
 
+_BatchNormOptions = collections.namedtuple(
```

Comment:
Make this ExperimentalBatchNormOptions, export it, and get rid of the method below. You can set the defaults for a namedtuple using e.g. https://stackoverflow.com/questions/11351032/namedtuple-and-default-values-for-optional-keyword-arguments or inheritance which would also allow you to set the docstring.

Reply:
(This was from TF API review.)